policies/project-director-election-process.md

```diff
@@ -0,0 +1,93 @@
+    * The facilitator may specify a minimum or maximum time period
+* Share nominations
+    * *Nominations from all teams will be pooled together. In other words, teams do not have to nominate a set of directors as a whole. They may nominate any number of potential directors and the Council will select from the pool a number of directors equal to the number of open seats.*
+    * Each team representative must share their team's nominations and the reasoning for their nominations first with the candidates being nominated, and then, for those candidates who consent to be nominated, with the council's selected facilitator.
+    * The facilitator must publicly share the pool of all nominations and reasonings at least ten days before the final election meeting.
+        * This allows other teams to share feedback or objections about candidates their team didn't initially consider with their representative.
+    * This 10-day period is equivalent to the Change Round in the "Election Meeting" step; representatives may change their nominations in advance of the meeting, this does not require restarting the 10-day period.
+
+### Election Meeting
+
+The election meeting is comprised of all members of the Leadership Council who will actively participate in the consensus process as well as the facilitator who will facilitate that process.
+
+* Share nominations
+    * Write down nominations first[^1].
+    * Use a round format[^2] (e.g. [round format](https://www.sociocracyforall.org/on-rounds/))
+    * Nominations must include supporting reasoning 
+* Change Round
+    * During the change round, every team member will share their current preferred nominees and the reasons for changes from the previous preferred nominees if that applies.
+    * Note that the change round is driving around discussing and trying to select the best possible nominee(s) rather than discussing the full set of nominees. De-selecting a nominee from consideration in a given change round does not disqualify the nominee from being selected in future change rounds.
+* Facilitator proposes a candidate:
+    * Based on gathered data, the facilitator suggests a candidate they believe will receive the group's consent.
+* Seek objections to the facilitator's proposal
+* Finalize the selection:
+    * Discuss and attempt to resolve any objections, if possible.
+    * Consent has been reached if the proposed choice has no remaining objections! :tada:
+    * If objections persist and the current proposal reaches an impasse, the facilitator should make a new proposal based on the discussion and feedback received.
+    * Continue seeking consensus and addressing objections until a satisfactory candidate is selected.
+        * Note that this process seeks to produce a candidate everyone is satisfied with and has no objections to. Avoid lengthy debates over preferences not related to objections or unmet requirements.
+* Document and announce the selection
```

Comment:
```suggestion
        * Note that this process seeks to produce a candidate everyone is satisfied with and has no objections to. Avoid lengthy debates over preferences not related to objections or unmet requirements.
* At this point, the Council will conduct a vote of the members of the council to ratify the selection through an election.
* Document and announce the selection
```

@rylev, @Mark-Simulacrum - Does this addition look sufficient to address to legal questions around needing a formal election?

Reply:
@eholk - @rylev and @Mark-Simulacrum have some wording that has been approved by Foundation's legal council.

Reply:
I added the edits from legal last week, so I'll mark this as resolved.